<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="m_general">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>通用</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
          <item>
           <widget class="QGroupBox" name="m_desktopClock">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>桌面时钟</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="m_editClock">
                 <property name="text">
                  <string>调整位置</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QCheckBox" name="m_showGanzhi">
                 <property name="text">
                  <string>显示干支年</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="m_showLunar">
                 <property name="text">
                  <string>显示农历</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>大小</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_clockSize">
                 <property name="minimum">
                  <number>100</number>
                 </property>
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="value">
                  <number>300</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_format">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>格式</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="verticalWidget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>格式说明</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableInfo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="rowCount">
                <number>9</number>
               </property>
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>30</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <column/>
               <column/>
               <item row="0" column="0">
                <property name="text">
                 <string>年</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>yy yyyy</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>月</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string>M MM mmm mmmm</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>日</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="2" column="1">
                <property name="text">
                 <string>d dd</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="3" column="0">
                <property name="text">
                 <string>时</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="3" column="1">
                <property name="text">
                 <string>h H hh HH</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="4" column="0">
                <property name="text">
                 <string>分</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="4" column="1">
                <property name="text">
                 <string>m mm</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="5" column="0">
                <property name="text">
                 <string>秒</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="5" column="1">
                <property name="text">
                 <string>s ss</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="6" column="0">
                <property name="text">
                 <string>干支</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="6" column="1">
                <property name="text">
                 <string>GG SS</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="7" column="0">
                <property name="text">
                 <string>星期</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="7" column="1">
                <property name="text">
                 <string>ddd dddd</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="8" column="0">
                <property name="text">
                 <string>上下午</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="8" column="1">
                <property name="text">
                 <string>a ap</string>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>例如: 2023-01-08 09:05:01</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="m_formatEdit">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>日期格式，如: yyyy-MM-dd</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_applyButton">
              <property name="text">
               <string>应用</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="m_preview">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="holiday">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>假期</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>年</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>4</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_addYear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>增加年</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_delYear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>删除年</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableDay">
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_addDay">
              <property name="text">
               <string>增加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_delDay">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
